tests/csm/rest/test_capacity_quota.py

```diff
@@ -1279,3 +1279,270 @@ def test_41156(self):
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41968')
+    def test_41968(self):
+        """
+        Test IO operations on suspended IAM user
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Creating IAM user with suspended False")
+        uid = "iam_user_1_" + str(int(time.time()))
+        bucket_name = "iam-user-bucket-" + str(int(time.time()))
+        tenant = "tenant_" + system_utils.random_string_generator()
+        self.log.info("Creating new iam user  %s", uid)
+        payload = self.csm_obj.iam_user_payload_rgw("loaded")
+        payload.update({"uid": uid})
+        payload.update({"tenant": tenant})
+        payload.update({"display_name": uid})
+        payload.update({"suspended": False})
+        resp1 = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify Response : %s", resp1)
+        assert_utils.assert_true(resp1.status_code == HTTPStatus.CREATED,
+                                 "IAM user creation failed")
+        user_id = resp1.json()['tenant'] + "$" + uid
+        self.created_iam_users.add(user_id)
+        resp = self.csm_obj.compare_iam_payload_response(resp1, payload)
+        assert_utils.assert_true(resp[0], resp[1])
+        akey = resp1.json()["keys"][0]["access_key"]
+        skey = resp1.json()["keys"][0]["secret_key"]
+        self.log.info("Step 2: Create bucket under above IAM user")
+        self.log.info("Verify Create bucket: %s with access key: %s and secret key: %s",
+                      self.bucket, akey, skey)
+        bucket_created = s3_misc.create_bucket(bucket_name, akey, skey)
+        assert bucket_created, "Failed to create bucket"
+        self.log.info("Step 3: Perform IOs")
+        resp = s3_misc.create_put_objects(self.obj_name, bucket_name,
+                                          akey, skey)
+        assert resp, "Put object Failed"
+
+        self.log.info("Step 4: Update Suspended True for same IAM User")
+        payload.update({"suspended": True})
+        response = self.csm_obj.modify_iam_user_rgw(user_id, payload)
+        assert response.status_code == HTTPStatus.OK, \
+                  "Status code check failed for updating iam user."
+        self.log.info("Step 5: Create bucket under above IAM user")
+        self.log.info("Verify Create bucket: %s with access key: %s and secret key: %s",
+                      self.bucket, akey, skey)
+        try:
+            bucket_created = s3_misc.create_bucket(bucket_name)
+            assert_utils.assert_false(bucket_created, "Failed to create bucket")
+        except Exception as error:
+            self.log.info("Expected exception received %s", error)
+        self.log.info("##### Test ended -  %s #####", test_case_name)
```

Comment:
Update to False again and check if the buckets are passing

Reply:
We had decided to remove this part from TD if i am right

Reply:
added scenario for Suspended False